easy-connect.h

```diff
@@ -67,43 +69,61 @@ NetworkInterface* easy_connect(bool log_messages = false) {
     int connect_success = -1;
 #if MBED_CONF_APP_NETWORK_INTERFACE == WIFI_ESP8266
     if (log_messages) {
-        output.printf("[EasyConnect] Using WiFi (ESP8266) \r\n");
-        output.printf("[EasyConnect] Connecting to WiFi..\r\n");
+        printf("[EasyConnect] Using WiFi (ESP8266) \n");
+        printf("[EasyConnect] Connecting to WiFi..\n");
+    }
+    connect_success = wifi.connect(MBED_CONF_APP_WIFI_SSID, MBED_CONF_APP_WIFI_PASSWORD, NSAPI_SECURITY_WPA_WPA2);
+    network_interface = &wifi;
+#elif MBED_CONF_APP_NETWORK_INTERFACE == WIFI_ODIN
+    if (log_messages) {
+        printf("[EasyConnect] Using WiFi (ODIN) \n");
+        printf("[EasyConnect] Connecting to WiFi..\n");
     }
-    connect_success = esp.connect(MBED_CONF_APP_WIFI_SSID, MBED_CONF_APP_WIFI_PASSWORD);
-    network_interface = &esp;
+    connect_success = wifi.connect(MBED_CONF_APP_WIFI_SSID, MBED_CONF_APP_WIFI_PASSWORD, NSAPI_SECURITY_WPA_WPA2);
+    network_interface = &wifi;
 #elif MBED_CONF_APP_NETWORK_INTERFACE == ETHERNET
     if (log_messages) {
-        output.printf("[EasyConnect] Using Ethernet\r\n");
+        printf("[EasyConnect] Using Ethernet\n");
     }
     connect_success = eth.connect();
     network_interface = &eth;
 #endif
 #ifdef MESH
     if (log_messages) {
-        output.printf("[EasyConnect] Using Mesh\r\n");
-        output.printf("[EasyConnect] Connecting to Mesh..\r\n");
+        printf("[EasyConnect] Using Mesh\n");
```

Comment:
If we set up the serial properly, you don't have to fool around with the \r\n. There's an easy way, did it on the same go for the mbed-os-example-client PR - https://github.com/ARMmbed/mbed-os-example-client/pull/176.

Adding this to mbed_add.json fixes it. Should we add it already here? 

            "platform.stdio-baud-rate":115200,
            "platform.stdio-convert-newlines":true,